app/views/components/_markdown-editor.html.erb

```diff
@@ -0,0 +1,52 @@
+<%
+  label_text ||= "Enter your markdown here"
+  textarea ||= {}
+  error_items ||= []
+   component_classes = %w[app-c-markdown-editor govuk-form-group]
+  component_classes << "govuk-form-group--error" if error_items.any?
+   textarea[:id] = textarea[:id].present? ? textarea[:id] : textarea[:name]
```

Comment:
You're using this check in couple of places. Can it be extracted somewhere? Is `textarea` only defined in the view?

Reply:
We don't need all this logic if we're explicit about the ID. Simplified the code.